setup.cfg

```diff
@@ -1,2 +1,4 @@
 [flake8]
 max-line-length=119
+# extend-ignore uses flake8's default ignore list, and adds to it
+extend-ignore = E501
```

Comment:
Can we doc somewhere why we are adding this exception, and maybe discuss if we should? I tried to start this at https://opendataservices.gitbook.io/developer-docs/code/code-python#coding-style previously - otherwise it's really hard for new dev's to work out why we do certain things

Reply:
Fair point. In this case it's related to a new formatting tool I was trying out ("black"). On reflection, I've decided to not make this change to the flake8 config now, and we can have a separate discussion about the new tool.

Reply:
Thanks. I've heard of Black before and would be very interested to hear more, but that should def be a separate team discussion.